src/Uplink/API/Validation_Response.php

```diff
@@ -496,15 +497,15 @@ public function to_wp_format() {
 		if ( empty( $this->response->auth_required ) || $this->resource->has_valid_auth_token( (array) $this->response->origin ) ) {
 			$info->download_link = isset($this->response->download_url) ? $this->response->download_url . '&pu_get_download=1' : '';
 		} else {
-			$url 		  = $this->response->origin->url;
+			$url          = $this->response->origin->url;
 			$query_params = [
-				'callback_uri' => urlencode( sprintf( '%s/stellarwp/connect', get_site_url() ) ),
+				'callback_uri' => urlencode( sprintf( '%s/%s', get_site_url(), Namespaces::get_hook_name( 'connect', '%TEXTDOMAIN%' ) ) ),
```

Comment:
This is hard to understand what's going on. I'm not sure I understand why the `Namespaces` class is necessary?